src/components/sidemenu/SideMenuPanel.js

```diff
@@ -156,6 +158,7 @@ const getMenuItems = ({ API, hideSidemenu, showDialog, navigation, store, theme
       name: 'Delete wallet',
       color: 'red',
       action: () => {
+        fireEventByCode(CLICK_DELETE_WALLET)
```

Comment:
@StanislavShevchenko did you implement the logout event triggering from the dashboard when the current route is Dashboard/Delete?

Reply:
@yaroslav-fedyshyn-nordwhale can you elaborate please

Reply:
@StanislavShevchenko why aren't you communicating on this?

Reply:
@sirpy @StanislavShevchenko We have this route on Dapp - Dashboard/Delete which is also opening the delete popup. (https://prnt.sc/ps4lvb) I asked if the delete event pushed to analytics in this case.

Reply:
@yaroslav-fedyshyn-nordwhale 
This code calls a unified deletion method where analytics has already been added

Reply:
@StanislavShevchenko you are right. thanks